.gitignore

```diff
@@ -18,3 +18,4 @@ cypress/screenshots
 cypress/videos
 /config/finance/json-data/*
 *.mdx
+*storybook.log
```

Comment:
This change shouldn't be part of this PR.

Reply:
Removed